lib/config/migrations/migrations-service.ts

```diff
@@ -151,7 +152,7 @@ export class MigrationsService {
     DatasourceMigration,
     RecreateClosedMigration,
     StabilityDaysMigration,
-    FetchReleaseNotesMigration,
+    FetchChangeLogsMigration,
```

Comment:
Revert

Reply:
Could you please explain why?
1. we rename variable
2. we need to migrate data from the old format to the new

Reply:
Because you are incorrectly removing an old migration which is still needed

Reply:
Agree with you.
Before I got another error. 
2 migrations for same property.

In this file line 186 -221 (Get Migrations functions)

Rename properties migration has more priority than custom migration.

In case you have an old configuration.
{
fetchReleaseNotes:"true"
}
then first will rename property
{
fetchChangeLogs: "true"
}
then we need to change "true" to "PR" in custom migration and we got result
{
fetchChangeLogs: "PR"
}

Reply:
@rarkins @viceice Could you please verify this comment?

In case I revert these changes test will fail.